docs/repair.md

```diff
@@ -108,6 +108,12 @@ CKE decides to execute a repair operations if its `operation` matches `OPERATION
 When CKE executes the check command, it appends the IP address of the target machine to the command.
 The command should return a string `true` if it evaluates the machine as healthy.
 
+`success_command` and its timeout are used when all of the repair steps are successfully finished.
```

Comment:
This description seems misleading.
CKE in this PR will execute `success_command` when it finds that `health_check_command` returns `true`.
* CKE will abort the sequence of the repair steps if the machine gets healthy; not all of the repair steps are executed.
* Even when all of the repair steps finish successfully, the machine may still be unhealthy.

```suggestion
`success_command` and its timeout are used when the machine is evaluated as healthy and the repair operation finishes successfully.
```